src/pyhf/interpolators/code0.py

```diff
@@ -71,8 +71,11 @@ def __call__(self, alphasets):
             'sa,shb->shab', alphasets, self.deltas_dn
         )
 
-        masks = tensorlib.einsum(
-            'sa,shb->shab', where_alphasets_positive, self.broadcast_helper
+        masks = tensorlib.astensor(
```

Comment:
doesn't einsum return a tensor anyways?

Reply:
@lukasheinrich It does. Dumb question: Do we support a way to cast the return Tensor from `tensorlib`?